pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: lints
-description: Official Dart and Flutter lint rules
-version: 0.1.0
+description: Official Dart and Flutter lint rules.
```

Comment:
Remove the Flutter part since that's now separate?

Reply:
(also in the readme?)

Reply:
Totally.  Good catch.